app/policies/contact_request_policy.rb

```diff
@@ -0,0 +1,5 @@
+class ContactRequestPolicy < AdminPolicy
+  def index?
+    admin_or_converter?
```

Comment:
Let's just have this be `admin?` The converters only need to deal with the Source Files and Data Imports.

Reply:
@littleforest Thanks! I'll use the defaults from AdminPolicy, which exactly do that